### ISSUE TITLE
Dupe glitch with Filing Cabinet and Simple Storage

### ISSUE THREAD

**Issue by wortzinator** [Darkosto/SkyFactory4Issues#495]
I was trying to automate the hopper Bonsai pots using Real Filing Cabinets and Simple Storage. I was doing some testing and found that if you lock the Filing cabinet and then try to pull an item out of a file, the item is duped. If you click on a item that has a quantity, it will dupe whatever quantity is in the file. While the Cabinet is still locked you cannot put the items back in the file, but once you unlock the cabinet you can put the items in the system.

Steps to replicate:

1) have a Real Filing Cabinet filing cabinet setup with a file folder inside (lets say a stack of Diamond blocks)
2) Attach a storage link cable, storage network master, and storage request table from Simple Storage.

3)Lock the filing cabinet
4)Click on the stack of 64 Diamond blocks.

Every time you click on the blocks you get 64 in your inventory.

5)Unlock the filing cabinet and put the 64 back in to make 128.
6)Repeat steps 3 and 4. This time you get 128 blocks of Diamond every time you click.
7)Continue this process till you have 10,000 (whatever number really) blocks of Diamonds.

In short,
When you view a file inside a locked filing cabinet using a storage request table from Simple Storage. You will dupe the entire amount of items that is in the folder.

**Comment by observeroftime02** [comment]
Good catch. It seems like that when the filing cabinet is locked, the simple storage network is able to pull items without actually decreasing the amount inside the filing cabinet. You're not able to put items back into the cabinet using the simple storage network when it's locked, however. You must unlock it first.

**Comment by Delerium** [comment]
Wanted to add that this dupe behavior also happens with a filter set on a real filing cabinet.  Whatever item is not selected by the filter gets duped.  I assume this is related to locking the filing cabinet, so this looked like the best place to post it.
